lib/frameworks/cucumber.js

```diff
@@ -1,14 +1,15 @@
-var ConfigParser = require('../configParser'),
-  q = require('q');
+var q = require('q');
 
 /**
  * Execute the Runner's test cases through Cucumber.
  *
  * @param {Runner} runner The current Protractor Runner.
  * @param {Array} specs Array of Directory Path Strings.
+ * @param {function(string): string} resolvePath resolves a path relative to the
+ *     config file
  * @return {q.Promise} Promise resolved with the test results
  */
-exports.run = function(runner, specs) {
+exports.run = function(runner, specs, resolvePath) {
```

Comment:
Seams reasonable but we will have to export the config file's directory so cucumber can do its logic.